src/pages/en/tutorial/3-components/1.md

```diff
@@ -87,18 +89,70 @@ This allows you to get started quickly with any working code, often duplicated t
 
 ### Test your knowledge
 
-Use these words in the text below. Click the blanks to reveal if you're right!
 
-<Blanks>
-- reusable
-- `src/pages`
-- refactor
-- standard HTML elements
-</Blanks>
-
-`.astro` files can use || **standard HTML elements** || and other Astro components in the same template, alongside each other. This means that you can decide when and where to || **refactor** || your code, creating || **reusable** || fragments of HTML defined in an Astro component. 
-
-These components should live in a folder outside of || **`src/pages`** || since they are not meant to be pages on your website.
+1. Astro files can use these alongside Astro components in their template:
+  <MultipleChoice>
+    <Option>
+      reusable
+    </Option>
+    <Option>
+      `src/pages/`
+    </Option>
+    <Option>
+       refactor
+    </Option>
+    <Option isCorrect>
+       standard HTML elements
+    </Option>
+  </MultipleChoice>
```

Comment:
Trimming down to 3 options and… maybe being too silly with the options?

```suggestion
1. Astro files can use these alongside Astro components in their template:
    <MultipleChoice>
      <Option>
        too much JavaScript
      </Option>
      <Option>
        `src/pages/`
      </Option>
      <Option isCorrect>
        standard HTML elements
      </Option>
    </MultipleChoice>
```

Reply:
I like being silly! But, they probably CAN use too much JS?  😅 

I'll think of another silly one!